### DISCUSSION TITLE
What did I do wrong?

### DISCUSSION THREAD

**Discussion by Morazee** [decaporg/decap-cms#6484] [General]
![image](https://user-images.githubusercontent.com/100149661/170832788-d8993ebc-c5ab-44aa-9c3b-22e3d39aa5b4.png)

**Comment by bradystroud** [comment]
Hey @Morazee 
Try these steps https://stackoverflow.com/questions/18419144/npm-not-working-read-econnreset?answertab=trending#tab-top
